types/onoff.d.ts

```diff
@@ -0,0 +1,42 @@
+
+declare type Directions = "in" | "out";
+declare type Edges = "none" | "rising" | "falling" | "both";
+declare type GpioOptions = {
+    debounceTimeout: number,
+    activeLow: boolean,
+    reconfigureDirection: boolean,
+}
+declare type BinaryValue = 0 | 1;
+declare type ValueCallback = (err: Error | undefined, value: BinaryValue) => void;
+
+declare class Gpio {
+    static HIGH: number;
+    static LOW: number;
+    static accessible: boolean;
+    constructor(gpio: number, direction: Directions | "high" | "low", edge: Edges);
+
+    read(callback: ValueCallback): void;
+    read(): Promise<BinaryValue>;
+
+    readSync(): BinaryValue;
+
+    write(value: BinaryValue, callback: (err?: Error) => void): void;
+    write(value: BinaryValue): Promise<void>;
+
+    writeSync(value: BinaryValue): void;
+
+    watch(callback: ValueCallback): void;
+    unwatch(callback: ValueCallback): void;
+    unwatchAll(): void;
+
+    direction(): Directions;
+    setDirection(direction: Directions): void;
```

Comment:
I think the [documentation for setDirection](https://github.com/fivdi/onoff#setdirectiondirection) may be incorrect and that the valid values for direction are 'in', 'out', 'high' and 'low'. I'll have to try it out on real hardware to verify this.

Reply:
That would be great as I was a little confused by this as well.

Reply:
I just tested it on hardware. The documentation for setDirection is incorrect and has been corrected.